spec/duration.html

```diff
@@ -1380,24 +1382,26 @@ <h1>
             1. Set _oneYearMonths_ to _untilResult_.[[Months]].
         1. Else if _largestUnit_ is *"month"*, then
           1. Let _moveResult_ be ? MoveRelativeDate(_calendar_, _relativeTo_, _oneMonth_).
-          1. Set _relativeTo_ to _moveResult_.[[RelativeTo]].
+          1. Let _newRelativeTo_ be _moveResult_.[[RelativeTo]].
```

Comment:
The loops for `"month"` and `"week"` don't actually need to be changed, because `relativeTo` isn't read after the loops. I assume you just did this for consistency?

Reply:
I overlooked that it wasn't necessary. Well, we could undo these changes for the month and week loops, but I think having it consistent is probably good anyway.